react/features/dial-out/components/DialOutNumbersForm.web.js

```diff
@@ -139,12 +139,11 @@ class DialOutNumbersForm extends Component {
      */
     render() {
         const { t, _dialOutCodes } = this.props;
-        const items
-            = _dialOutCodes ? this._formatCountryCodes(_dialOutCodes) : [];
 
         return (
             <div className = 'form-control'>
-                { this._createDropdownMenu(items) }
+                { _dialOutCodes ? this._createDropdownMenu(
```

Comment:
Do you want to handle the case now where the array is empty or would it be preferable to allow the blank list of dial codes to display?

Also, how about returning null instead of an empty space? Maybe null is more explicit that nothing should be returned from the ternary?